client/src/app/kommissions/view/kommission-view.component.html

```diff
@@ -16,7 +16,12 @@ <h1>{{ kommission?.name }}</h1>
           <h2>Effectif</h2>
         </section>
       <p>
-        Les barmen actifs de la kommission sont: <span *ngFor="let barman of getBarmen(true)">{{ barman.nickname }}</span>
+        Les barmen actifs de la kommission sont: 
+        <span *ngFor="let barman of getBarmen(true)">
+          <a [routerLink]="'/barmen/' + barman.id">
+            {{ barman.nickname }}, 
```

Comment:
There will be one more comma at the end of the line, you can use `barmens.join(',') + '.'` instead

Reply:
That's a tips :) Thanks !

Reply:
This tips is a better one if we want to keep 'a' tag :) https://stackoverflow.com/a/34053861